source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -30,7 +30,7 @@
 # www.navitia.io
 
 from __future__ import absolute_import, print_function, unicode_literals, division
-import logging
+import logging, flask, json, hashlib, datetime
 from flask import request, g
```

Comment:
Why import flask? Only flask.request seems to be used and it's already imported.

Reply:
:+1: removed